themes/default/content/resources/getting-started-with-azure-native/index.md

```diff
@@ -50,11 +50,11 @@ main:
     # URL for embedding a URL for ungated webinars.
     youtube_url: ""
     # Sortable date. The datetime Hugo will use to sort the webinars in date order.
-    sortable_date: 2021-04-27T09:00:00-07:00
+    sortable_date: 2021-04-28T09:00:00-07:00
     # Duration of the webinar.
     duration: "1 hour"
     # Datetime of the webinar.
-    datetime: "APR 27TH, 2021"
+    datetime: "APR 28TH, 2021"
```

Comment:
Nit, but should we title-case these months now that we're no longer doing the all-caps thing?

Reply:
```suggestion
    datetime: "April 28th, 2021"
```